src/Workspaces/Remote/Core/ProjectSystem/WorkspaceProjectFactoryServiceDescriptor.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Remote.ProjectSystem
+{
+    internal static class WorkspaceProjectFactoryServiceDescriptor
```

Comment:
❓ Why did this need to move to the workspaces layer? I can't find any references to it outside the VS layer, and the service it exports isn't present except in the VS layer.